js/data/utils.js

```diff
@@ -153,14 +153,22 @@ function isConjunctiveOperator(condition) {
     return /^(and|&&|&)$/i.test(condition);
 }
 
+function getKeyValue(keyName, key) {
+    let value;
+    keyName.split('.').forEach((name) => {
+        value = value ? value[name] : key[name];
+    });
+
+    return value;
+}
+
 const keysEqual = function(keyExpr, key1, key2) {
```

Comment:
The `keyExpr` variable was not used in a comparison since the method was written (4c89bcaff145 in hg). @AlekseyMartynov , can you explain, why?